packages/rum-core/test/performance-monitoring/transaction.spec.js

```diff
@@ -76,13 +76,6 @@ describe('transaction.Transaction', function() {
     tr.end()
   })
 
-  it('should store context.page.url', function() {
-    var tr = new Transaction('/', 'transaction')
-    tr.detectFinish()
-    var location = tr.context.page.url
-    expect(location).toBe(window.location.href)
-  })
```

Comment:
Isn't this test still relevant since we add the context in transaction service?

Reply:
Let me revert this. Thought its not super useful.

Reply:
We handle it as part of the transaction service end transaction. I am removing this from here since there are already tests  in transaction service and also for context related tests.

Reply:
Do we have a test that checks the context to have the `window.location.href` value?  which one are you referring to ?